### PR TITLE
ONE 4 mythics 

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/sword_of_forge_and_frontier.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Sword of Forge and Frontier
+ManaCost:3
+Types:Artifact Equipment
+K:Equip:2
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 2 | AddToughness$ 2 | AddKeyword$ Protection from red & Protection from green | Description$ Equipped creature gets +2/+2 and has protection from red and from green.
+T:Mode$ DamageDone | ValidSource$ Creature.EquippedBy | ValidTarget$ Player | CombatDamage$ True | Execute$ ExileTwo | TriggerZones$ Battlefield | TriggerDescription$ Whenever equipped creature deals combat damage to a player, exile the top two cards of your library. You may play those cards this turn. You may play an additional land this turn.
+SVar:ExileTwo:DB$ Dig | Defined$ You | DigNum$ 2 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ exile the top two cards of your library. You may play those cards this turn. You may play an additional land this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sword_of_forge_and_frontier.txt:7]: No SpellDescription

File: forge-gui/res/cardsfolder/upcoming/sword_of_forge_and_frontier.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Sword of Forge and Frontier
+ManaCost:3
+Types:Artifact Equipment
+K:Equip:2
+S:Mode$ Continuous | Affected$ Creature.EquippedBy | AddPower$ 2 | AddToughness$ 2 | AddKeyword$ Protection from red & Protection from green | Description$ Equipped creature gets +2/+2 and has protection from red and from green.
+T:Mode$ DamageDone | ValidSource$ Creature.EquippedBy | ValidTarget$ Player | CombatDamage$ True | Execute$ ExileTwo | TriggerZones$ Battlefield | TriggerDescription$ Whenever equipped creature deals combat damage to a player, exile the top two cards of your library. You may play those cards this turn. You may play an additional land this turn.
+SVar:ExileTwo:DB$ Dig | Defined$ You | DigNum$ 2 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ STPlay | SubAbility$ DBCleanup | ForgetOnMoved$ Exile | Duration$ EndOfTurn | SubAbility$ DBEffectBis
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ Until the end of your next turn, you may play the exiled cards.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/sword_of_forge_and_frontier.txt:9]: `EffectZone$ Command` shouldn't be needed, this is set by the Effect anyway

**Comment by Simisays** [comment]
Seems like copying a planeswalker spell from exile results in those planeswalkers entering the battlefield with no loyalty. 
Copying planeswalker does seem to work correctly with CopySpellAbility

**Review by tool4ever — DISMISSED** [review]

File: forge-gui/res/cardsfolder/upcoming/capricious_hellraiser.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+SVar:TrigExile:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | AtRandom$ True | Hidden$ True | ChangeType$ Card.YouOwn | ChangeNum$ 3 | RememberChanged$ True | SubAbility$ DBChoose
+SVar:DBChoose:DB$ ChooseCard | ChoiceZone$ Exile | Amount$ 1 | Choices$ Card.IsRemembered+nonLand+nonCreature | ChoiceTitle$ Choose a noncreature, nonland card | SubAbility$ DBPlay
+SVar:DBPlay:DB$ Play | Defined$ ChosenCard | ValidSA$ Spell | Controller$ You | CopyCard$ True | WithoutManaCost$ True | ValidSA$ Spell | Optional$ True | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Graveyard
+DeckHints:Ability$Mill
+Oracle:This spell costs {3} less to cast if you have nine or more cards in your graveyard.Flying\nWhen Capricious Hellraiser enters the battlefield, exile three cards at random from your graveyard. Choose a noncreature, nonland card from among them and copy it. You may cast the copy without paying its mana cost.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/capricious_hellraiser.txt:14]: graveyard.\nFlying

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**